ui/pages/notifications-settings/notifications-settings.tsx

```diff
@@ -0,0 +1,189 @@
+import React, { useEffect, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import type { InternalAccount } from '@metamask/keyring-api';
+import { useI18nContext } from '../../hooks/useI18nContext';
+import { useMetamaskNotificationsContext } from '../../contexts/metamask-notifications/metamask-notifications';
+import {
+  NOTIFICATIONS_ROUTE,
+  DEFAULT_ROUTE,
+} from '../../helpers/constants/routes';
+import {
+  Box,
+  IconName,
+  Text,
+  ButtonIcon,
+  ButtonIconSize,
+} from '../../components/component-library';
+import {
+  BlockSize,
+  BorderColor,
+  Display,
+  JustifyContent,
+  FlexDirection,
+  AlignItems,
+  TextVariant,
+  TextColor,
+} from '../../helpers/constants/design-system';
+import { NotificationsPage } from '../../components/multichain';
+import { Content, Header } from '../../components/multichain/pages/page';
+import {
+  selectIsMetamaskNotificationsEnabled,
+  selectIsMetamaskNotificationsFeatureSeen,
+  getIsUpdatingMetamaskNotifications,
+  getIsUpdatingMetamaskNotificationsAccount,
+} from '../../selectors/metamask-notifications/metamask-notifications';
+import { getInternalAccounts } from '../../selectors';
+import { NotificationsSettingsAllowNotifications } from './notifications-settings-allow-notifications';
+import { NotificationsSettingsTypes } from './notifications-settings-types';
+import { NotificationsSettingsPerAccount } from './notifications-settings-per-account';
+
+// Define KeyringType interface
+type KeyringType = {
+  type: string;
+};
+
+// Define AccountType interface
+type AccountType = InternalAccount & {
+  balance: string;
+  keyring: KeyringType;
+  label: string;
+};
+
```

Comment:
I've seen this in a few places. Is this intentional? Can we reuse an existing type?